src/libraries/Directory.Build.props

```diff
@@ -304,9 +308,12 @@
 
   <PropertyGroup Condition="'$(BuildAllConfigurations)' != 'true'">
     <!-- We add extra binplacing for the test shared framework until we can get hardlinking with the runtime directory working on all platforms -->
-    <BinPlaceTestSharedFramework Condition="'$(BuildTargetFramework)' == '$(NetCoreAppCurrent)'">true</BinPlaceTestSharedFramework>
+    <BinPlaceTestSharedFramework Condition="$(BuildTargetFramework)' == '$(NetCoreAppCurrent)'">true</BinPlaceTestSharedFramework>
+    
     <BinPlaceNETFXRuntime Condition="'$(BuildTargetFramework)' == '$(NetFrameworkCurrent)'">true</BinPlaceNETFXRuntime>
 
+    <BinPlaceTestRuntimePack Condition="'$(TargetOS)' == 'iOS' or '$(TargetOS)' == 'tvOS' or '$(TargetOS)' == 'Android'">true</BinPlaceTestRuntimePack>
```

Comment:
It seems like we're starting to use these long condition in a lot of places: `'$(TargetOS)' == 'iOS' or '$(TargetOS)' == 'tvOS' or '$(TargetOS)' == 'Android'` -- should we introduce a property to make it easier? Something like `TargetsMobile`?

Reply:
I was wondering the same thing.

Reply:
I think it would make sense to do it now before we start adding it to more places. Looks like we should be able to define it here https://github.com/dotnet/runtime/blob/07965e57e294493757bb14f274137d7047628d3b/src/libraries/targetframework.props in the according `when` statements.

Reply:
Do we want to replace all instances of `'$(TargetOS)' == 'iOS' or ...` with a `TargetsMobile` property? Defining it in `targetframework.props` and replacing all instances of that condition everywhere seems to break things. Replacing only in `src/installer` and `src/libraries` builds w/o error, but we then lose a couple folders `lib-runtime-packs/data` `lib-runtime-packs/runtimes/$(PackageRID)/native`.

Reply:
We can do it in a separate PR to not complicate things here.